app/models/concerns/callable.rb

```diff
@@ -30,5 +30,15 @@ def contacted_since(datetime)
       { since: datetime, contacts: patients_reached.length,
         first_contacts: first_contact.length, pledges_sent: 20 }
     end
+
```

Comment:
Spare code from when I was determining inactive via a lack of recent call activity. Not used in archive process.